lab_1_classify_by_unigrams/main.py

```diff
@@ -5,107 +5,62 @@
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
-
-def tokenize(text: str) -> list[str] | None:
-    """
-    Split a text into tokens.
-
```

Comment:
it is forbidden to delete docstrings, return all back!

Reply:
done